docs(readme): add tip to configure luarocks install with envvars (#131)
nvim-neorocks/rocks.nvim: README.md

> For security reasons, we recommend that you read `:help :source`
> and the installer code before running it so you know exactly what it does.

> [!TIP]
>

> > In order to configure luarocks installation to use a specific lua install,

Reviewer: ```suggestion
> To configure the luarocks installation to use a specific lua install,
```
